src/themes/teams/components/List/listItemVariables.ts

```diff
@@ -1,4 +1,7 @@
 export default siteVariables => ({
+  // Root
+  backgroundColor: siteVariables.brand,
```

Comment:
nit: just to prevent confusion (and for the sake of being more explicit about the use case), we may consider to rename it to something like `activeBackgroundColor` or `backgroundActiveColor` - as it would better suggest when this color will be applied